src/models/Project/Project.ts

```diff
@@ -14,11 +15,17 @@ export interface ProjectsResponse {
     id: string;
     location: Address;
     name: string;
+    client?: {
+        id?: string;
+        companyName?: string;
+    }
 }
 
-export interface CreateProjectResponse {
+export interface CreateOrUpdateProjectResponse {
     description: string;
     id: string;
     location: Address;
     name: string;
+    clientId: string | null;
```

Comment:
Probably better to just use the `?` here unless you will specifically be setting clientId to null

Reply:
I got it